examples/official/trial/mnist_pytorch/distributed.yaml

```diff
@@ -9,8 +9,7 @@ hyperparameters:
   dropout1: 0.25
   dropout2: 0.5
 resources:
-  # Use 16 GPUs to train the model.
-  slots_per_trial: 16
+  slots_per_trial: 8
```

Comment:
What's the reason for this change? Just curious, seems reasonable because this is what the tests run anyways (`test_pytorch_parallel`).

Reply:
The quick start tutorial references this file but claims that `slots_per_trial` is 8. Seems like a reasonable default value anyway...